packages/ui/src/theme/css/component/breadcrumbs.scss

```diff
@@ -15,6 +15,16 @@
   line-height: var(--amplify-components-breadcrumbs-item-line-height);
   color: var(--amplify-components-breadcrumbs-item-color);
   font-size: var(--amplify-components-breadcrumbs-item-font-size);
+  &--disabled {
+    cursor: not-allowed;
+    .amplify-breadcrumbs__link {
+      background-color: var(
+        --amplify-components-breadcrumbs__link-disabled-color
+      );
+      color: var(--amplify-components-breadcrumbs__link-disabled-color);
+      pointer-events: none;
+    }
```

Comment:
Is this needed here? Should be covered by lines 45-51.

Reply:
I guess it depends, the customer who filed the bug report mentioned wanting the ability to disable `Link` or `Item` so this allows you to pass the prop to either. But if we only want the prop to work with `Item` then this isn't needed.

Reply:
Actually think I misunderstood, so without this you'd need to add `isDisabled` to `Breadcrumsb.Link` in addition to `Breadcrumsb.Item` otherwise the link would still be active. 

IMO it makes sense the way it is because you can also create the component like `<Breadcrumbs items=...` so it might be confusing if you could pass `isDisabled` in with `items` in only that case but you'd need to put it in both `Link` and `Item` if using `Breadcrumbs.Items`